packages/controller/src/types.ts

```diff
@@ -104,6 +104,7 @@ export interface Keychain {
   signMessage(
     typedData: TypedData,
     account: string,
+    sync?: boolean,
```

Comment:
hmm I think we need to invert this to make it backward compatible, since there are existing controller libraries in the wild. so undefined / false should have the existing behavior
```suggestion
    async?: boolean,
```

Reply:
Oh that's true indeed. When designing this I only considered "getting something signed" as the keychain's public contract, not that "some UI is displayed". That's why I didn't consider changing the behavior here a breakage.

I can update to flip the behavior tho.

Reply:
Updated to flip to default.